properties.schema

```diff
@@ -19,6 +19,15 @@
       "inputType": "Text",
       "validators": [],
       "translatable": true
+    },
+    "popupPagination": {
+      "type": "string",
+      "required": true,
+      "default": "Slide {{itemNumber}} of {{totalItems}}.",
```

Comment:
might be an idea to set this to `"{{itemNumber}} /  {{totalItems}}"` so that the current pagination format is maintained (seeing as the Grunt task will automatically add this to course.json)